src/Flysystem/FlysystemAssetStore.php

```diff
@@ -1459,6 +1461,23 @@ public static function flush()
     }
 
     public function getResponseFor($asset)
+    {
+        /** @var HTTPResponse $response */
+        /** @var array $context */
+        [$response, $context] = $this->generateResponseFor($asset);
```

Comment:
That's a shorthand for `list($response, $context) = $this->generateResponseFor($asset);`

Reply:
This seems over-complicated - should we just use `list()` to make it clearer? I'm not fussed either way...

Reply:
It's a valid PHP syntax that meet our minimum PHP version. 

My general view is that we shouldn't hold back using new PHP feature if they meet our minimum PHP requirements ... otherwise you get into a chicken and egg problem where you can't use new syntax because people aren't familiar with it and people are not familiar with the new syntax because you're not using it.

Reply:
Yeah, the array syntax (instead of `list()`) is pretty standard nowadays.